src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/CommandBarTests/UnoSamples_Tests.NativeCommandBar.cs

```diff
@@ -27,5 +27,40 @@ public void NativeCommandBar_Automated()
 
 			_app.WaitForText(result, "Clicked!");
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.Android)]
+		public void NativeCommandBar_Content_Alignment_Automated()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.CommandBar.CommandBar_Native_With_TextBox");
+
+			var verticalValue = _app.Marked("verticalValue");
+			var horizontalValue = _app.Marked("horizontalValue");
+			var innerTextBox = _app.Marked("InnerTextBox");
+			var innerTextBlock = _app.Marked("InnerTextBlock");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'innerTextBlock'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5212257)